meshapp/src/components/Header.tsx

```diff
@@ -7,11 +7,15 @@ import React from "react";
 
 const Header = () => {
   return (
-    <AppBar position="static" sx={{ background: "#0B7D66" }}>
+    <AppBar
+      position="static"
+      sx={{ backgroundColor: "primary.main" }}
+      elevation={0}
+    >
       <Toolbar>
         {/*---Left Side Menu Elements (Menu/Notification Icon)---*/}
         <Grid container>
-          <IconButton sx={{ mr: 2, color: "#0000008F" }}>
+          <IconButton sx={{ mr: 2, color: "secondary.main" }}>
```

Comment:
the color should be "action.active", you can see the difference if you switch to light mode

Reply:
👍